tests/contrib/celery/test_integration.py

```diff
@@ -426,3 +429,68 @@ def fn_task_parameters(user, force_logout=False):
         assert dd_span.get_tag('celery.id') == t.task_id
         assert dd_span.get_tag('celery.action') == 'apply_async'
         assert dd_span.get_tag('celery.routing_key') == 'celery'
+
+
+class CeleryDistributedTracingIntegrationTask(CeleryBaseTestCase):
+    """Distributed tracing is tricky to test for two reasons:
```

Comment:
👍 thanks for the test and great accompanying documentation too! Much appreciated 😄